### PR TITLE
fix: add uid in evt

### LINKED ISSUE #9065: [CAL-1745] Different Booking links generated in seated event calendar event and email
Found a bug? Please fill out the sections below. 👍

### Issue Summary

Private discussion:- https://threads.com/34476359991?s=7GGqabSnMtjJXWqesAUNJY

### Steps to Reproduce

1. connect google calendar and turn on offer seats in event type settings and add slots
2. book a meeting and you would receive correct meeting url in calendar event in google
3. when another user books the same time slot then the calendar event description would get updated even though the url has not changed in email or booking page.

we get wrong meeting url in the calendar event

Any other relevant information. For example, why do you consider this a bug and what did you expect to happen instead?

### Technical details

- Browser version, screen recording, console logs, network requests: You can make a recording with [Bird Eats Bug](https://birdeatsbug.com/).
- Node.js version
- Anything else that you think could be an issue.

<sub>[CAL-1745](https://linear.app/calcom/issue/CAL-1745/different-booking-links-generated-in-seated-event-calendar-event-and)</sub>

### PR DESCRIPTION
**Pull request by Udit-takkar**
## What does this PR do?

Fixes  https://github.com/calcom/cal.com/issues/9065

Now we get same meeting url in email, calendar event and booking page 

Screenshot 2023-05-23 at 5 36 58 PM

Screenshot 2023-05-23 at 5 37 20 PM

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by joeauyeung — CHANGES_REQUESTED** [review]
I just tested this and the location data in the calendar description becomes false.

image

**Review by joeauyeung — APPROVED** [review]
Tested with Cal video and Meet. Working as expected.

**Comment by Udit-takkar** [comment]
@joeauyeung no idea what caused this bug

File: packages/lib/CalEventParser.ts [hunk 1]
```diff
@@ -194,7 +194,7 @@ ${getAppsStatus(calEvent, t)}
 ${
   // TODO: Only the original attendee can make changes to the event
   // Guests cannot
-  !calEvent.seatsPerTimeSlot && getManageLink(calEvent, t)
+  calEvent.seatsPerTimeSlot ? "" : getManageLink(calEvent, t)
```
Comment (Udit-takkar) [inline packages/lib/CalEventParser.ts:197]: @joeauyeung this check was printing false in the calendar event description. 

Now fixed
Screenshot 2023-05-23 at 6 34 47 PM

**Pull request merged**